src/objects/aff/zcl_abapgit_json_handler.clas.abap

```diff
@@ -79,7 +79,7 @@ CLASS zcl_abapgit_json_handler DEFINITION
         IMPORTING it_defaults TYPE ty_skip_paths
         CHANGING  co_ajson    TYPE REF TO zcl_abapgit_ajson
         RAISING   zcx_abapgit_ajson_error,
-    set_custom_enum_deserialize
+      set_custom_enum_deserialize
```

Comment:
@larshp abapLint didn't complain about the indentation here. Maybe a small bug?

Reply:
the https://rules.abaplint.org/in_statement_indentation/ rule is not enabled in abapGit, https://abaplint.app/stats/abapGit/abapGit/disabled_rules

Reply:
ok, thanks